doc/troubleshooting.rst

```diff
@@ -209,7 +206,7 @@ If your Python version is correct and you still have the same error, then try on
        docker run --rm -e LOCAL_USER_ID=`id -u $USER` -v $(pwd):/opt/data -p 8080:8080 nestsim/nest:<version> notebook
 
 Can't find an answer to your question?
-----------------------------------------------
+--------------------------------------
 
 We may have answered your question on GitHub or in our Mailing List!
```

Comment:
Suggest to make these hyperlinks.

Reply:
That's a good suggestion! I updated the hyperlinks.